src/bootstrap/native.rs

```diff
@@ -789,6 +789,17 @@ impl Step for Lld {
             .define("LLVM_CONFIG_PATH", llvm_config_shim)
             .define("LLVM_INCLUDE_TESTS", "OFF");
 
+        if builder.config.llvm_thin_lto {
+            cfg.define("LLVM_ENABLE_LTO", "Thin");
+            if !target.contains("apple") {
+                cfg.define("LLVM_ENABLE_LLD", "ON");
+            }
+        }
+
+        if let Some(ref linker) = builder.config.llvm_use_linker {
+            cfg.define("LLVM_USE_LINKER", linker);
+        }
```

Comment:
This is duplicated from the `Llvm` step. Can we add a function that applies the LLVM config to both Llvm and Lld so we ensure they stay in sync?

Reply:
Move the code into its own helper function as requested